_data/simple-icons.json

```diff
@@ -4044,6 +4044,11 @@
             "source": "https://partnermarketinghub.withgoogle.com/",
             "guidelines": "https://about.google/brand-resource-center/brand-elements/"
         },
+        {
+            "title": "Google AdMob",
+            "hex": "EA4335",
+            "source": "https://cdn.worldvectorlogo.com/logos/google-admob.svg"
```

Comment:
Good catch! Yes, I think we should use the Wikipedia one here

Reply:
I agree, but I have a doubt.
I've made some changes in the official logo to look more beautiful. Because if I just get the SVG file like it will look like this:

![admob](https://user-images.githubusercontent.com/16006294/164998363-dacb0555-598f-4854-8ffd-c0ba0c0f9931.png)

What do you think is better to do?

Reply:
Thanks for the reply, I will make the change today and then ask you to review! 🙌